domain/servicedefinition/servicedefinition.go

```diff
@@ -53,7 +53,7 @@ type ServiceDefinition struct {
 	HealthChecks      map[string]domain.HealthCheck // HealthChecks for a service.
 	Prereqs           []domain.Prereq               // Optional list of scripts that must be successfully run before kicking off the service command.
 	MonitoringProfile domain.MonitorProfile         // An optional list of queryable metrics, graphs, and thresholds
-	MemoryLimit       float64
+	MemoryLimit       int64
```

Comment:
Do we need to worry about any kind of type conversion when the user upgrades from 1.0.x to 1.1?